src/plugins/procmon/procmon.cpp

```diff
@@ -174,8 +174,9 @@ static char* read_cmd_line(vmi_instance_t vmi, drakvuf_trap_info_t* info, addr_t
                 for(i = 0;i<cmd_len;i++)
                 {
                     strncat(cmd,&buf_ret[i],1);
-                }
+                }                
             }
+            g_free(buf_ret);
```

Comment:
@tklengyel

Reply:
If this will return `NULL`:
https://github.com/tklengyel/drakvuf/pull/833/files#diff-e1df45a792096383aa407daca2f1f8acR171

then the line you've mentioned will never be reached and thus `buf_ret` will leak.